src/main/java/com/pirimid/cryptotrade/util/CoinbaseUtil.java

```diff
@@ -20,52 +24,97 @@
 import java.util.ArrayList;
 import java.util.Base64;
 import java.util.List;
-import java.util.Locale;
 
 @Component
 public class CoinbaseUtil {
 
 
-   public static String getSignature(String timestamp,String secretKeyString,String method,String path,String body) throws NoSuchAlgorithmException, InvalidKeyException {
-        String prehash = timestamp+method+path+body;
+    public static String getSignature(String timestamp, String secretKeyString, String method, String path, String body) throws NoSuchAlgorithmException, InvalidKeyException {
+        String prehash = timestamp + method + path + body;
         byte[] secretKeyDecoded = Base64.getDecoder().decode(secretKeyString);
         SecretKey secretKey = new SecretKeySpec(secretKeyDecoded, "HmacSHA256");
         Mac hmacSha256 = Mac.getInstance("HmacSHA256");
         hmacSha256.init(secretKey);
-        String sin =  Base64.getEncoder().encodeToString(hmacSha256.doFinal(prehash.getBytes()));
+        String sin = Base64.getEncoder().encodeToString(hmacSha256.doFinal(prehash.getBytes()));
 
-       return sin;
+        return sin;
     }
 
-    public static PlaceOrderResDTO getPlaceOrderResDTO(PlaceOrderResCoinbaseDTO placeOrderResCoinbaseDTO){
-        PlaceOrderResDTO placeOrderResDTO;
+    public static OrderResDTO getPlaceOrderResDTO(PlaceOrderResCoinbaseDTO placeOrderResCoinbaseDTO) {
+        OrderResDTO orderResDTO;
         Gson gson = new Gson();
         String orderCoinbaseres = gson.toJson(placeOrderResCoinbaseDTO);
-        placeOrderResDTO = gson.fromJson(orderCoinbaseres,PlaceOrderResDTO.class);
-        placeOrderResDTO.setExecutedAmount(placeOrderResCoinbaseDTO.getExecuted_value());
-        placeOrderResDTO.setType(OrderType.valueOf(placeOrderResCoinbaseDTO.getType().toUpperCase()));
-        placeOrderResDTO.setSide(Side.valueOf(placeOrderResCoinbaseDTO.getSide().toUpperCase()));
-        placeOrderResDTO.setStatus(Status.valueOf(placeOrderResCoinbaseDTO.getStatus().toUpperCase()));
-        placeOrderResDTO.setCreatedAt(placeOrderResCoinbaseDTO.getCreated_at());
-        placeOrderResDTO.setSymbol(placeOrderResCoinbaseDTO.getProduct_id());
-        return placeOrderResDTO;
+        orderResDTO = gson.fromJson(orderCoinbaseres, OrderResDTO.class);
```

Comment:
Can we have @builder or constructor here?

Reply:
ok, will add @Builder everywhere instead of setter.